include/boost/spirit/home/classic/utility/scoped_lock.hpp

```diff
@@ -12,7 +12,7 @@
 #include <boost/spirit/home/classic/namespace.hpp>
 #include <boost/thread/lock_types.hpp>
 #if !defined(BOOST_SPIRIT_COMPOSITE_HPP)
```

Comment:
@djowel do you have any idea why there is that `#if`?  It predates migration from CVS, as well as the include guard in the included file.

Reply:
I have no idea, or perhaps I already forgot.

Reply:
I am going to make the include unconditional then, cannot imagine why it would be wrong.